powershell-adapter/Tests/native_and_powershell.dsc.yaml

```diff
@@ -21,3 +21,6 @@ resources:
     keyPath: HKLM\Software\Microsoft\Windows NT\CurrentVersion
     valueName: ProductName
     _ensure: Present
+metadata:
```

Comment:
`dsc` will inject this into the config so shouldn't have it explicitly here unless there is some reason?

Reply:
didn't know

Reply:
it does make testing adapters tricky because you have to just know that this happens. I favor the convenience, but we should make sure this is documented in the future.